### DISCUSSION TITLE
How to control packages listed under an individual (`Config/runiverse/noindex: true` not working)

### DISCUSSION THREAD

**Discussion by TimTaylor** [r-universe-org/help#422] [Q&A]
I'm trying to ensure that I control the packages that are listed on https://timtaylor.r-universe.dev/packages but I'm somewhat struggling.

I'm using `Config/runiverse/noindex: true` in the DESCRIPTION (pushed to CRAN) but it only seems to be working for packages under https://github.com/TimTaylor/ not those I maintain but in other repositories. It has stopped rebuilding on new commits / releases but it has not cleared out the old packages. Is there something else I should be doing?

**Comment by jeroen** [comment]
Thanks, I'll have a look. I noticed the job is complaining: https://github.com/r-universe/reconverse/actions/workflows/sync.yml but maybe we haven't implemented the part to delete packages that are already in the universe, once they gain such a tag.

Btw what is the reason you don't want the software to be discoverable on r-universe?

**Reply by TimTaylor** [reply]
Great, cheers.

> Btw what is the reason you don't want the software to be discoverable on r-universe?
Two main reasons:

- Predominantly as it makes users think it's curated by myself as opposed to scraped and built which can be confusing. Rebuilding CRAN packages and hosting the binaries is of course ok but the additional maintainer linkage seems unneeded. If linking is to be done I'd prefer it defaulted to release builds, not the main branch, to at least ensure it matches CRAN.
- I agree with Henrik's [wish on the ordering of repos](https://github.com/r-universe-org/help/discussions/375#discussion-6321877)

To clarify, I like the r-universe but, for me, more as a tool when needed not as a CRAN-alternative. My use cases are/will be: (a) sharing non-CRAN packages to colleagues or (b) for CI needs (e.g. recently I needed a non-GitHub devel package available to the r-lib/actions so used r-universe for that).

Hope that makes sense/helps.
